templates/search_page.html

```diff
@@ -136,10 +137,19 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                     <div class="flex-fill">
                         <input class="form-control border-0" type="search" id="location" placeholder="Search"
-                            aria-label="Search" name="location" value="Los Angeles, CA, USA">
+                            aria-label="Search" name="location" value="Los Angeles, CA, USA" onchange="fetchLocation(this)"> 
                     </div>
 
-                    <div class="fixedwidthicon">
+                    <script>
+                        function fetchLocation() {
+                            const loc = document.getElementById('location');
+                            if (loc) {
+                                sessionStorage.setItem("locationPerm", loc.value)
+                                console.log(`The location is ${loc.value}`)
+                            }
+                        }
+                    </script>
+                            <div class="fixedwidthicon">
```

Comment:
plz also consider using HTML/JS formatter to align the indentation.
e.g. <div class="fixedwidthicon"> has a different indent

Reply:
addressed at 7c58805 , I fixed the indentation for `search_page.html`